SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier1/DetritiveTrematode.cs

```diff
@@ -83,22 +83,16 @@ public void OnDamageDealtServer(DamageReport report)
                 //1 = 30%
                 //5 = 65%
                 // 9 = 105%
-                if (victim.combinedHealthFraction < requiredHealthPercentage && !hasDot && report.damageInfo.dotIndex != _trematodeDotIndex && (victim.gameObject != attacker))
+                if (victim.combinedHealthFraction < requiredHealthPercentage && !hasDot && (victim.gameObject != attacker))
                 {
 
-                    // do the first tick instantly
-                    DamageInfo damageInfo = new DamageInfo();
-                    damageInfo.attacker = base.gameObject;
-                    damageInfo.crit = false;
-                    damageInfo.damage = trematodeDamage * base.body.damage;
-                    damageInfo.force = Vector3.zero;
-                    damageInfo.inflictor = base.gameObject;
-                    damageInfo.position = victim.body.corePosition;
-                    damageInfo.procCoefficient = 0f;
-                    damageInfo.damageColorIndex = DamageColorIndex.Item;
-                    damageInfo.damageType = DamageType.DoT;
-                    damageInfo.dotIndex = _trematodeDotIndex;
-                    victim.TakeDamage(damageInfo);
+                    // do the first "tick" instantly
+                    FuckinUhhhhhhhhhhOrb orb = new FuckinUhhhhhhhhhhOrb();
```

Comment:
Utilize a proper TypeName for this orb, what even is a "Fuckin Uhhhhhh Orb"?

Reply:
wrong of you but I fixed it